internal/api/handlers/clientdata.go

```diff
@@ -1,9 +1,19 @@
 package handlers
 
 import (
+	"bytes"
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"io"
 	"log"
 	"net/http"
 	"regexp"
+	"strconv"
+
+	"github.com/pluque01/CofreSagradoVirtual/internal/api/utils"
```

Comment:
¿No era una buena práctica en go definir en go.mod nombres cortos para las bibliotecas internas? ¿O incluso hacerlo aquí directamente?

Reply:
Por lo que yo sé y por lo que he estado investigando no se pueden definir otros nombres para las bibliotecas, ni para las internas ni para las externas.
https://stackoverflow.com/questions/61405061/module-name-alias-in-go-mod